RecoTauTag/RecoTau/plugins/PFRecoTauLostTrackPlugin.cc

```diff
@@ -0,0 +1,83 @@
+/*
+ * =============================================================================
+ *       Filename:  PFRecoTauLostTrackPlugin.cc
+ *
+ *    Description: Add references to tracks of tau-charged-hadrons built on 
+ *                 top of a track
+ *
+ *        Created:  25/04/2022
+ *
+ *         Authors:  Michal Bluj (NCBJ, Warsaw)
+ *
+ * =============================================================================
+ */
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+#include "RecoTauTag/RecoTau/interface/RecoTauBuilderPlugins.h"
+#include "DataFormats/TauReco/interface/PFTau.h"
+#include "DataFormats/TauReco/interface/PFRecoTauChargedHadron.h"
+#include "DataFormats/TauReco/interface/PFRecoTauChargedHadronFwd.h"
```

Comment:
This does not seem to be used and can be removed, unless you want to use the definition of `PFRecoTauChargedHadronCollection` at [L59](https://github.com/cms-sw/cmssw/pull/37918/files#diff-51256b8c93e7a78492ef9842cc03a30a51d35de4ad31c02f0ff1e9749cff2ccdR59)